samples/net/http_server/README.rst

```diff
@@ -0,0 +1,266 @@
+.. _http_server:
+
+HTTP server
+###########
+
+The HTTP server sample demonstrates how to host an HTTP server that runs on an Internet connected Nordic device via cellular or Wi-Fi.
+
+.. include:: /includes/net_connection_manager.txt
+
+.. contents::
+   :local:
+   :depth: 2
+
+Requirements
+************
+
+The sample supports the followiURLng development kits:
```

Comment:
```suggestion
The sample supports the following development kits:
```

Reply:
^^  @divipillai  Could you include this suggestion?